.flake8

```
@@ -1,17 +1,24 @@
 # Taken directly from https://github.com/ambv/black/blob/master/.flake8
 [flake8]
 ignore =
-    C408, # ignore temporarily https://github.com/adamchainz/flake8-comprehensions/pull/282
+    B905,
+    # ignore temporarily https://github.com/adamchainz/flake8-comprehensions/pull/282
```

This is not temporary anymore.
```suggestion
    # allow calling dict() instead of the literal {} for readability
```